pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+# Merging the Berkeley schema into the NMDC schema
+
+_Corresponding to a jump from major version 10 to 11_
+
+This major refactoring increases the number of **classes** in the schema from 70 to 75, but decreases the number of root
+classes from 19 to 17 due to better organization.
+
+| Removed legacy class                | New/Replacement class from Berkeley schema | Selected legacy parents   |
+|-------------------------------------|--------------------------------------------|---------------------------|
+| Activity                            |                                            |                           |
```

Comment:
I can read these OK (no issues). I noticed occurrences of these strings below the table are formatted `list this` (with backticks), whereas they are not formatted that way in this table. I don't know whether adding backticks here would make it more or less difficult for readers to grok.

Reply:
no, we do not want to add backticks here. that would break the pretty table rendering

Reply:
I think GitHub renders inline backtick-ed strings within tables OK; e.g.:

| A | B |
|--|--|
| bare | bare |
| `wrapped` | `wrapped` |
| bare | bare |
| `wrapped` | `wrapped` |
| bare | bare |
| `wrapped` | `wrapped` |